### ISSUE TITLE
Essentials Sign

### ISSUE THREAD

**Issue by philman247** [EssentialsX/Essentials#1193]
I'm having a problem with akk of the "Buy" "Sell" Signs, they don't work when deopped, (default group). And i've given the permissions essentials.signs.use.* and others. Any ideas? when i right click a sign it stays blank. (doesnt say nothing)

**Comment by Motorloid01** [comment]
Do you have them enabled in the config

**Comment by CCCoderDE** [comment]
I have the same problem.

But I made an interesting experience...
I was using my Non-OP-Account and it didn't work. But when my OP-Account was standing in front of the sign and I used my Non-OP-Account to "click" on the sign **through the body** of my OP-Account, it worked.

Very strange but thats my current situation...

Already tried it with a third account from a friend and the same thing happens on his computer. H has to click through my character to make it work...

**Comment by CCCoderDE** [comment]
And yes, they are enabled in the config (my OP-Account can use them) and everyone has the Permission to use them (they work when clicking through).

**Comment by CCCoderDE** [comment]
Ok so I found that the problem was the spawn-protection in the server.properties-File.... Maybe that is your Problem as well @philman247 

**But I still think this needs to be fixed** You should be able to use Essentials-Signs inside the Spawn-Protection as many Servers put them in the Spawn....

**Comment by SupaHam** [comment]
That protection is provided by Minecraft itself. Although I haven't tested it, I would guess that it's not something we should fix or override anyways to maintain expected behaviour.

If you want signs that close to spawn, you can set `spawn-protection` to 0 and create a region using WorldGuard.

**Comment by CCCoderDE** [comment]
@SupaHam It still doesn't make any sense that I can use the signs in the spawn-area when clicking through another player....

**Comment by CCCoderDE** [comment]
This is definitly is not the way it should be..

**Comment by SupaHam** [comment]
It is, as designed by Minecraft. If you want to prevent the vanilla behaviour you *MUST* set `spawn-protection` to a lower value that does not intersect with the sign location.

**Comment by CCCoderDE** [comment]
I definitly understand this and agree on you, that I should not be able to use these signs in spawn-protection. 

**But why can I use them when clicking "through" the body of another player...?**

**Comment by SupaHam** [comment]
Ah, that I'm not sure of. This must be caused by another plugin or minecraft. I don't believe Essentials

**Comment by SupaHam** [comment]
This is produced in [SignPlayerListener]( https://github.com/drtshock/Essentials/blob/2.x/Essentials/src/com/earth2me/essentials/signs/SignPlayerListener.java#L38). I think a valid solution would be implementing a check of that spawn radius or remove the if statement entirely?

Reasons for removing the if statement would be because we should let the client handle it, things like the block border are reactive features that communicate with the user to tell them they can use the EssentialsX feature.

**Comment by pop4959** [comment]
This issue is really hard to follow. It sounds like the original author had a permission issue, but things got derailed. If there is a relevant bug, it should probably be made into it's own issue so we can track it better.
